SingularityExecutor/src/main/java/com/hubspot/singularity/executor/config/SingularityExecutorConfiguration.java

```diff
@@ -678,6 +681,15 @@ public void setCronDirectory(String cronDirectory) {
     this.cronDirectory = cronDirectory;
   }
 
+  public boolean isUseFileAttributes() {
```

Comment:
names like `isxxx()` and `getxxx()` are compliant with the JavaBeans spec for naming getter methods for booleans. That way, things that use reflection (like jackson) can find teh boolean getters. If we want it to read differently, we can renamed the field to something more compatible with `is<Field Name>`